### PR TITLE
Enable all stable features on the playground

### PR DESCRIPTION
**Pull request by Nemo157**
Prompted by [this u.rl.o thread](https://users.rust-lang.org/t/error-unresolved-import-futures-threadpool/34469), based on docs [in the playground source](https://github.com/integer32llc/rust-playground/blob/f68374b402b5dedbcea417086af29deb315e3031/top-crates/src/main.rs#L183-L192).

### CODE REVIEW

**Comment by Nemo157** [comment]
I assume that the playground builds all crates together in a single build so activating the features from the facade will affect all other crates and there's no point in marking them individually.

File: futures/Cargo.toml [hunk 1]
```diff
@@ -56,3 +56,6 @@ read-initializer = ["futures-io/read-initializer", "futures-util/read-initialize
 
 [package.metadata.docs.rs]
 all-features = true
+
+[package.metadata.playground]
+all-features = false
```
Comment (taiki-e) [inline futures/Cargo.toml]: The title says 'Enable', but I think '= false' means disable 'all-features'.

**Comment by taiki-e** [comment]
also, some features depend on unstable features and require nightly rustc, so I think it is good to include only stabilized features.

**Comment by Nemo157** [comment]
@taiki-e thanks, that's a good point. I've listed out just the stable features instead.

File: futures/Cargo.toml [hunk 2]
```diff
@@ -56,3 +56,6 @@ read-initializer = ["futures-io/read-initializer", "futures-util/read-initialize
 
 [package.metadata.docs.rs]
 all-features = true
+
+[package.metadata.playground]
+features = ["std", "async-await", "compat", "io-compat", "executor", "threadpool"]
```
Comment (taiki-e) [inline futures/Cargo.toml]:
```suggestion
features = ["std", "async-await", "compat", "io-compat", "executor", "thread-pool"]
```

I renamed 'threadpool' feature to 'thread-pool' in #1937: https://github.com/rust-lang-nursery/futures-rs/pull/1937/files#diff-41c03002d828b5290c4f39adc8d88060R45

**Pull request merged**